Descent3/init.cpp

```diff
@@ -1993,82 +1866,51 @@ void InitD3Systems1(bool editor) {
   //	load the string table
   InitStringTable();
 
-  if (!IsLocalOk()) {
-#ifdef WIN32
-    MessageBox(NULL, "Sorry, your computer has a language not supported by this version of Descent 3.", PRODUCT_NAME,
-               MB_OK);
-#else
-    printf("Sorry, your computer has a language not supported by this version of Descent 3.");
-#endif
-    exit(0);
-  }
-
-// With 1.5 no more copy protection!
-#if 0
-	//CD Check goes here
-//#if ( defined(RELEASE) && (!defined(DEMO)) && (!defined(GAMEGAUGE)) )
-	if( (!FindArg("-dedicated")) )
-		PreGameCdCheck();
-#endif
-
-#ifdef LASERLOCK
-  // At this point the laser locked CD MUST be inserted or the
-  // game and possibly the entire OS will crash because laserlock
-  // will assume it's a hacked CD.
-  if (!ll_Init())
-    exit(0);
-  if (!ll_Verify())
-    exit(0);
-#else
-  bool CheckCdForValidity(int cd);
-  if ((!FindArg("-dedicated"))) {
-    if (!CheckCdForValidity(CD_inserted)) {
-#ifdef WIN32
-      ShowCursor(true);
-      MessageBox(NULL, "Invalid CDROM!", PRODUCT_NAME, MB_OK);
-      ShowCursor(false);
-#endif
-      exit(0);
-    }
-  }
-#endif
   INIT_MESSAGE(("Initializing GFX"));
   InitGraphics(editor);
 
   //	initialize data structures
+  INIT_MESSAGE(("Initializing data structures."));
```

Comment:
Every init message in this function after `InitGraphics` crashes the game on Linux, probably because the graphics are half-initialized at this point

Reply:
Does the existing message "Initializing I/O systems" also crash it?